src/devices/Common/MathExtensions.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Iot.Device
+{
+    /// <summary>
+    /// Implementations of some functions missing in older .NET versions
+    /// </summary>
+    public static class MathExtensions
+    {
+        /// <summary>
+        /// Returns val, limited to the range min-max (inclusive)
+        /// </summary>
+        public static double Clamp(double val, double min, double max)
```

Comment:
It doesn't need to happen now but it would be good to add a few unit tests to these members to make sure they behave as expected.

Reply:
Done